templates/contrib/forms/search.html

```diff
@@ -1,30 +1,34 @@
 {% load static %}
 {% load i18n %}
 {% load a4a %}
-<form class="fr-col-12 a4a-search-form fr-search-bar fr-search-bar--lg"
+<form class="fr-col-12 "
       id="search-form"
       role="search"
       action="{% if search_action %}{{ search_action }}{% else %}{% url 'search' %}{% endif %}">
     <div class="fr-mx-0 w-100">
         <div class="fr-grid-row fr-grid-row--gutters">
-            <div class="fr-col-12 fr-col-sm-3 fr-input-group input-form text-start fr-pr-1v">
+            <div class="fr-col-12 fr-col-lg-4">
                 <label for="what-input">
-                    <span class="larger-font{% if dark %} text-light{% endif %}"><strong>{% translate "Nom de l'établissement" %}</strong></span>
-                    <br>
+                    {% translate "Quel nom ?" %}
+                    <span class="fr-hint-text">{% translate "Quel est le nom de l’établissement ?" %}</span>
                 </label>
                 {% comment %}Translators: please do not translate "Le Louvre, Café de la gare" as we are searching into french contents{% endcomment %}
-                <input type="search" id="what-input" name="what" class="fr-input" placeholder="Le Louvre, Café de la gare..." aria-label="{% translate "Saisir le nom de l'établissement. Exemple : Le Louvre, Café de la gare" %}" autocomplete="off" required {% if "what" in request.GET %} value="{{ what|default:request.GET.what }}" {% else %} value="{{ what }}" {% endif %}>
+                <input type="search"  class="fr-input" id="what-input" name="what" placeholder="Le Louvre, Café de la gare..." aria-label="{% translate "Saisir le nom de l'établissement. Exemple : Le Louvre, Café de la gare" %}" autocomplete="off" required {% if "what" in request.GET %} value="{{ what|default:request.GET.what }}" {% else %} value="{{ what }}" {% endif %}>
```

Comment:
is it a kind of guideline ? (class, then id, then name)

Reply:
it's yolo guideline